certified-connectors/Nodefusion Portal/apiDefinition.swagger.json

```diff
@@ -78,6 +78,60 @@
         ],
         "x-ms-visibility": "important"
       }
+    },
+    "/organizations/{organizationId}/workservice": {
+      "get": {
+        "tags": [
+          "Organization",
+          "WorkServices",
+          "Work",
+          "Services",
+          "Get"
+        ],
+        "summary": "Get Work Services",
+        "description": "Acquire Organization's Work Services along with consumption estimate",
+        "operationId": "GetOrgWorkServices",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "organizationId",
+            "required": true,
+            "type": "string",
+            "format": "uuid"

```

Comment:
@hkusulja The 'x-ms-summary' property is required, and the 'description' property is required. Please add both.

Reply:
@Amjed-Ayoub I've updated the swagger and added the required properties.